style/client.css

```diff
@@ -178,17 +178,20 @@ button:disabled {
 }
 */
 .dark .button,
-.dark .tabbar a.button {
+.dark .tabbar a.button,
+.dark .popupmenu button.button {
```

Comment:
I guess another class could be added to all the other popupmenu buttons, then the rules for them can be made more specific to not include these two buttons?